data/mods/Medieval_Stuff/armor.json

```diff
@@ -58,7 +58,7 @@
         "encumbrance" : 10,
         "bashing" : 10,
         "flags" : ["VARSIZE", "STURDY"],
-        "coverage" : 80,
+        "coverage" : 70,
```

Comment:
80
Should actually be 90 or 95 to be worth it, but for now it can be just 80.
Hard hat is the first good helmet and it has 80% coverage, so no real, forged, helmet should go below that.